src/sass/accordion/_base.scss

```diff
@@ -38,18 +38,32 @@
 
   &__toggle-text {
     color: $color-black-500;
-    font-size: $ts-18;
-    font-weight: $font-weight-bold;
+    font-size: $ts-20;
+    font-weight: $font-weight-regular;
     line-height: 1.3;
     padding-right: $spacing-sm;
   }
 
   &__toggle-icon {
     color: $color-black-400;
     margin-left: auto;
+    position: relative;
```

Comment:
Instead of messing with the positioning here, we can make this a little more flexible and resilient by setting the actual icon element to display flex and align-items to center. That way no matter what the icon size is, it will be vertically centered in relation to the toggle text.

```diff
- position: relative;
- top: 4px;
+ display: flex;
+ align-items: center;
```